packages/dds/task-manager/src/taskManager.ts

```diff
@@ -647,7 +647,6 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
         content.forEach(([taskId, clientIdQueue]) => {
             this.taskQueues.set(taskId, clientIdQueue);
         });
-        this.scrubClientsNotInQuorum();
```

Comment:
This is exciting if we don't need it anymore!  Did you verify if this situation no longer occurs (e.g. with your fuzz test)?

Reply:
Yes! 😄